presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotColumnUtils.java

```diff
@@ -112,7 +111,7 @@ public static Type getPrestoTypeFromPinotType(FieldSpec field, boolean inferDate
         return new ArrayType(getPrestoTypeFromPinotType(field.getDataType()));
     }
 
-    public static Type getPrestoTypeFromPinotType(DataType dataType)
+    public static Type getPrestoTypeFromPinotType(FieldSpec.DataType dataType)
```

Comment:
- private
- we should keep `DataType` static imported rather than having `FieldSpec.DataType` here